### PR TITLE
Code Copy Theming Improvements

### PR DESCRIPTION
**Pull request by StirStudios**
- Small update to optimize the copy code experience, css.
- You can see what type of code snippet it is: ts, vue, etc
- Fixed card body missing import.

### CODE REVIEW

File: apps/docs/src/reference/icons.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Icons
 
+<div class="lead mb-5">
```
Comment (VividLemon) [inline apps/docs/src/reference/icons.md:3]: Make this an alert

File: apps/docs/src/components/HighlightCard.vue [hunk 2]
```diff
@@ -13,3 +13,67 @@
 <script setup lang="ts">
 import {BCard, BCardBody} from 'bootstrap-vue-next'
 </script>
+
+<style lang="scss">
```
Comment (VividLemon) [inline apps/docs/src/components/HighlightCard.vue:17]: Move this back to the layout file

**Review by VividLemon — CHANGES_REQUESTED** [review]

File: apps/docs/src/components/HighlightCard.vue (hunk 2 above)
Comment (StirStudios) [reply apps/docs/src/components/HighlightCard.vue]: Done!

File: apps/docs/src/reference/icons.md (hunk 1 above)
Comment (StirStudios) [reply apps/docs/src/reference/icons.md]: Done!

**Pull request merged**